sway-core/tests/sway_to_ir/get_storage_key.sw

```diff
@@ -1,16 +1,18 @@
 contract;
 
-struct Empty {}
+struct Foo {
+    foo: u32
```

Comment:
As discussed, please create an issue to investigate the need for this change and the issues with the IR parser.

Reply:
Yep, I don't think this has been done yet?

Reply:
I guess this particular change is no longer there but an issue about empty structs in the IR and how they work if they're passed around etc. would still be useful.